### PR TITLE
feat: Add drone-artifactory plugin

### PR DESCRIPTION
**Pull request by athieriot**
Add entry for drone-artifactory plugin

### CODE REVIEW

File: content/athieriot/drone-artifactory/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* `version` - Artifact version, default to value from Pom file
+* `repo_key` - Target repository key, default to 'libs-snapshot-local' if version contains 'snapshot', 'libs-release-local' otherwise
+* `files` - List of files to deploy
+* `force_upload` - Force upload if a file already exists
+
+All file paths must be relative to current project sources
+
+File paths are interpreted with [node-glob](https://github.com/isaacs/node-glob#glob-primer) and can contain things such as regex, or directory wildcards(./\*\*/\*.js)
+
+## Secrets
```
Comment (appleboy) [inline content/athieriot/drone-artifactory/index.md]: remove secret section.

Comment (athieriot) [inline content/athieriot/drone-artifactory/index.md]: Should I replace it with something else?

File: content/athieriot/drone-artifactory/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* **ARTIFACTORY_USERNAME** - corresponds to **username**
+* **ARTIFACTORY_PASSWORD** - corresponds to **password**
+
+It is highly recommended to put the **ARTIFACTORY_USERNAME** and **ARTIFACTORY_PASSWORD**
+into secrets so it is not exposed to users. This can be done using the drone-cli.
+
+```bash
+drone secret add --image=athieriot/drone-artifactory \
+    octocat/hello-world ARTIFACTORY_USERNAME kevinbacon
+
```
Comment (appleboy) [inline content/athieriot/drone-artifactory/index.md]: Drone remove `image` flag from latest version.

File: content/athieriot/drone-artifactory/index.md (hunk 1 above)
Comment (appleboy) [inline content/athieriot/drone-artifactory/index.md]: NO. just remove this section.

**Comment by bradrydzewski** [comment]
@athieriot can you structure your documentation similar to this:
https://raw.githubusercontent.com/drone/drone-plugin-index/master/content/drone-plugins/drone-s3/index.md

That way it is consistent with the other plugin documentation. 

For more details check out
http://readme.drone.io/tutorials/how-to-register-a-plugin/

**Comment by athieriot** [comment]
Sorry I didn't see that document. I updated the document accordingly

**Pull request merged**